product_variant_available_in_pos/models/product_product.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2016 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductProduct(models.Model):
+
+    _inherit = 'product.product'
+
+    available_in_pos = fields.Boolean(
+        string='Available in the Point of Sale',
+        help='Check if you want this product to appear in the Point of Sale')
+
+    @api.model
+    def create(self, vals):
+        product = super(ProductProduct, self).create(vals)
+        template_vals = {}
+        if 'available_in_pos' not in vals:
+            template_vals[
+                'available_in_pos'] = product.product_tmpl_id.available_in_pos
+        if template_vals:
+            product.write(template_vals)
+        return product
```

Comment:
This surely works, but I find it a bit confusing. May I suggest to rewrite it as:

```
    @api.model
    def create(self, vals):
        product = super(ProductProduct, self).create(vals)
        if 'available_in_pos' not in vals:
            product.available_in_pos = product.product_tmpl_id.available_in_pos
        return product
```

Reply:
Thanks for your review @espo-tony , I did not any refactoring because this is only a migration.
Anyway I added a new commit to add what you asked for: https://github.com/OCA/product-variant/pull/102/commits/83cc3ddb450f087e313212bce8ae0c4c57928967